jenkins-webhook/1.0.0/README.md

```diff
@@ -0,0 +1,12 @@
+# Jenkins Webhook Integration
+
+- Using Keptn version 0.10.* to 0.12.x? Use the [Keptn Jenkins Shared Library](https://artifacthub.io/packages/keptn/keptn-integrations/jenkins-library).
```

Comment:
```suggestion
- Using older Keptn version 0.10.x, 0.11.x and 0.12.x: Use the [Keptn Jenkins Shared Library](https://artifacthub.io/packages/keptn/keptn-integrations/jenkins-library).
```

Though in general, the webhook config you provide here most likely still can be used with older Keptn versions